pkg/vmimage/vmimage.go

```diff
@@ -165,7 +165,7 @@ func (builder *Builder) Run(ctx context.Context) error {
 
 	if builder.Validate {
 		builder.Log.Infof("copy file from VM")
-		err := builder.scp([]string{"/tmp/info", "/tmp/check"})
+		err := builder.scp([]string{"/tmp/info", "/tmp/check", "/tmp/scap-report.html"})
```

Comment:
It might also be worth copying over the raw logs of the scap run from `/tmp/oscap.log`

Reply:
The scap log has each of the checks performed logged to the screen.  If we feel it is worth it I can copy it.  It's the text output of run itself which results get written to the `scap-report.html`.

Reply:
Can we raise a follow-up issue to make sure we have integration/post run to publish these to ur private blob storage? This applies to info and checks results too?